roles/etcd/tasks/gen_certs.yml

```diff
@@ -34,28 +34,56 @@
 
 - name: Gen_certs | run cert generation script
   command: "{{ etcd_script_dir }}/make-ssl-etcd.sh -f {{ etcd_config_dir }}/openssl.conf -d {{ etcd_cert_dir }}"
+  environment:
```

Comment:
This still doesn't work correctly. It adds all nodes every time it decides to gen_certs.